share/functions/help.fish

```diff
@@ -71,7 +71,7 @@ function help --description 'Show help for the fish shell'
 		end
 	end
 
-	if test -z $fish_browser[1]
+	if set -q $fish_browser[1]
```

Comment:
You need to remove the `$`. Otherwise this looks good to me. If you fix this and confirm the resulting code works for you I'll merge it.